sharedbuffers/mapped_struct.py

```diff
@@ -2059,33 +2064,451 @@ def hinted_bsearch(a, hkey, hint):
                 raise NotImplementedError("Unsupported array type %s" % (chr(dtype),))
         finally:
             PyBuffer_Release(cython.address(indexbuf)) #lint:ok
+else:
+    import bisect
+    def _hinted_bsearch(a, hkey, hint):
+        return bisect.bisect_left(a, hkey)
+    globals()['hinted_bsearch'] = _hinted_bsearch
+
+#@cython.ccall
```

Comment:
They're informative. The declaration conflicts (ie: Cython doesn't let it be) with the one in the pxd. The pxd rules. But the pxd is invisible, so I left them commented for readers to know that this, is a ccall function. Without having to go to the pxd.